### DISCUSSION TITLE
Zombie project refuses to stop

### DISCUSSION THREAD

**Discussion by andydempster** [docksal/docksal#1592] [Q&A] [answered]
I've used a basic `fin init` to create a host for an [EspoCRM](https://github.com/espocrm/espocrm) demo.

Aside from adding some PHP overrides in `.docksal/etc/php/php.ini` it's a simple stack but is exhibiting some odd behaviour. It refuses to ~die~ stop!

`fin stop` shows the containers stopping but then 5 secs later they are back up. The same when stopping from Docker Desktop.

OSX 11.6.1 (Big Sur)
Docker Desktop 3.4.0 (65384)
fin version: 1.107.1

Can anyone shed some light on this?

**Comment by andydempster** [comment]
Note I added the PHP MyAdmin add on and this container is the only that stops when using Docker Desktop.

**Comment by lmakarov** [comment, marked as answer]
Congrats! You discovered a really awesome (but not widely known) Docksal feature - auto-starting projects upon HTTP request.

If you have a browser tab that's actively polling the web server, then Docksal will automatically wake up that project's stack.
You can disable this behavior with `fin config set --global PROJECT_AUTOSTART=0`.

See:

- https://docs.docksal.io/use-cases/project-auto-start-stop/
- https://docs.docksal.io/stack/configuration-variables/#project-autostart

Related issue: https://github.com/docksal/docksal/issues/1009

**Reply by lmakarov** [reply]
This feature makes most sense in the context of [Docksal Sandbox Server](https://github.com/docksal/sandbox-server) environments.
It probably makes more sense to keep it disabled by default for local environments and only enforced in sandbox stacks.
